qiskit/synthesis/permutation/permutation_full.py

```diff
@@ -74,6 +72,13 @@ def synth_permutation_acg(pattern):
 
     Returns:
         QuantumCircuit: the synthesized quantum circuit.
+
+    References:
+        1. N. Alon, F. R. K. Chung, and R. L. Graham.
+           *Routing Permutations on Graphs Via Matchings.*,
+           Proceedings of the Twenty-Fifth Annual ACM Symposium on Theory of Computing(1993).
+           Pages 583–591.
+           `10.1145/167088.167239 <https://www.cs.tau.ac.il/~nogaa/PDFS/r.pdf>`_
```

Comment:
Kind of neither here nor there, but it's weird to use a DOI as the link text but not the corresponding https://doi.org/10.1145/167088.167239 as the link target.

Reply:
true, I think I didn't even read what was in the link